Mu2eUtilities/src/ReSeedByEventID.cc

```diff
@@ -0,0 +1,35 @@
+#include "Offline/Mu2eUtilities/inc/ReSeedByEventID.hh"
+//#include "art/Framework/EventProcessor/Scheduler.h"
+
+#include "art/Framework/Services/Registry/ServiceHandle.h"
+
+#include "fhiclcpp/exception.h"
+
+#include <iostream>
+#include <string>
+
+namespace mu2e {
+  ReSeedByEventID::ReSeedByEventID( CLHEP::HepRandomEngine& engine, SeedService::seed_t salt):_salt(salt){
+    std::string engineType{art::ServiceHandle<art::RandomNumberGenerator>{}->defaultEngineKind()};
+    std::cout << "Kind:    " << engineType << std::endl;
+    if ( engineType != "MixMaxRng" ){
```

Comment:
Why is this restriction necessary?  it seems setSeeds is defined in the base class.

Reply:
Good catch.  Some time ago it was not in the base class and I did not check again.

Reply:
Actually, I need to check if all engines actually use all 4 seeds.  If some silently look at one only entry in the array then we need some sort of test, perhaps excluding this engines.  Perhaps alternate behaviour.

Reply:
I confirm that HepJamesRandom::setSeeds( const long*, int)  silently ignores all but the first seed.

https://gitlab.cern.ch/CLHEP/CLHEP/-/blame/develop/Random/src/JamesRandom.cc#L262

So the question is whitelist or blacklist?

Reply:
I think whitelisting MINMAX here is the right thing to do.
But why is the check done  on an object pulled from the service handle instead of using the constructor arg (engine)?